recipes/keystone/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+from conan.tools.build import check_max_cppstd
+import os
+
+required_conan_version = ">=2.0"
+
+
+class KeystoneConan(ConanFile):
+    name = "keystone"
+    description = (
+        "Keystone assembler framework: Core (Arm, Arm64, Hexagon, "
+        "Mips, PowerPC, Sparc, SystemZ & X86) + bindings."
+    )
+    license = ( "GPL-2.0-only", "DocumentRef-EXCEPTIONS-CLIENT" )
```

Comment:
```suggestion
    license = ( "GPL-2.0-only", "DocumentRef-EXCEPTIONS-CLIENT:LicenseRef-FOSS-License-Exception" )
```

As per the spdx syntax, you need to have a licenseRef here too